### PR TITLE
[TESTS] Added new atomic tests for Smoke phase 2

### PR DESCRIPTION
**Pull request by yevh-berdnyk**
status: ready

### CODE REVIEW

File: test/appium/tests/atomic/chats/test_public.py [hunk 1]
```diff
@@ -68,3 +68,53 @@ def test_public_chat_clear_history(self):
...
+class TestPublicChatSingleDevice(SingleDeviceTestCase):
+
+    @marks.testrail_id(3752)
+    def test_send_korean_characters(self):
+        sign_in = SignInView(self.driver)
+        home = sign_in.create_user()
+        chat_name = home.get_public_chat_name()
+        public_chat = home.join_public_chat(chat_name)
+        message = '파란하늘'
+        public_chat.chat_message_input.send_keys(message)
```
Comment (Serhy) [inline test/appium/tests/atomic/chats/test_public.py:115]: Well, one of the main points for this test is to check whether Message edit box could combine korean chars properly. So we need to send each char here, one by one: type`ㅎ` then`ㅏ` then`ㄴ` then `ㅡ` then `ㄹ`. Put cursor to the beginning of message text and type `ㅍ`,`ㅏ`,`ㄹ`,`ㅏ`,`ㄴ`. And after message sent - check if it's = `파란하늘`

**Review by Serhy — COMMENTED** [review]
Cool! But lets modify a bit test_send_korean_characters test.

File: test/appium/tests/atomic/account_management/test_profile.py [hunk 2]
```diff
@@ -192,3 +191,85 @@ def test_contact_profile_view(self):
...
+        public_key = home_2.get_public_key()
+        home_2.home_button.click()
+
+        profile_1 = home_1.profile_button.click()
+        profile_1.advanced_button.click()
+        profile_1.mail_server_button.click()
+        profile_1.plus_button.click()
+        server_name = 'test'
+        profile_1.specify_name_input.set_value(server_name)
+        profile_1.mail_server_address_input.set_value('enode://531e252ec966b7e83f5538c19bf1cde7381cc7949026a6e499b'
```
Comment (lukaszfryc) [inline test/appium/tests/atomic/account_management/test_profile.py]: I'd export the enode address to some constant

**Review by lukaszfryc — APPROVED** [review]

File: test/appium/tests/atomic/chats/test_public.py (hunk 1 above)
Comment (yevh-berdnyk) [reply test/appium/tests/atomic/chats/test_public.py:115]: I've skipped it for now, need more investigation here

**Review by Serhy — APPROVED** [review]

**Pull request merged**